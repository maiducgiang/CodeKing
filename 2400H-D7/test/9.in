10
48726 2471
29496 2499
33612 2438
13701 1111
47608 1206
33996 766
36880 966
8823 1585
25860 1661
43467 664
