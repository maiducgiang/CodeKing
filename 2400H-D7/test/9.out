293002347
146792041
481031386
428953463
268247046
35932396
446339431
874162636
313767831
470763085
